benchmarks/NNPDF_bench.py

```diff
@@ -35,6 +35,7 @@ def skip_pdfs(self, _theory):
 
 base_theory = {
     "Qref": 91.2,
+    "Qedref": 91.2,
```

Comment:
QED you can also spell uppercase ^^

Reply:
In any case, it is not the same `Q`, so also `Qqed` is fine, maybe even better. Just don't collapse the two Qs

Reply:
there is the theoretical constraint that the two things _have to be given at the same scale_, isn't it? so we don't need this (otherwise they can never evolve together ...)

Reply:
It's how it's called in banana. We have to change banana in order to change it

Reply:
Perfect, then change banana please :)
(unless is the name used by APFEL, check it - but I would change in any case, and translate for APFEL)

Reply:
@felixhekhorn yes but for some reason in banana the two Qref are separated. In eko I implemented it to be the same

Reply:
as said above: then banana is just wrong ;-) it is used nowhere else but in eko. I'd suggest to revert